src/mobile/testConfig.js

```diff
@@ -1,4 +1,6 @@
 import { configure } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
+import { i18nextInit } from 'shared-modules/libs/i18next';
 
 configure({ adapter: new Adapter() });
+i18nextInit().then(Promise.resolve());
```

Comment:
Is the `then` block necessary here?

Reply:
I'll check. Probably not

Reply:
Nope, it wasn't necessary. I'm not sure why I thought it was 🤔